isis/src/control/objs/ControlNet/ControlNet.h

```diff
@@ -329,18 +350,30 @@ namespace Isis {
 
     signals:
       void networkStructureModified();
+      void networkModified(ControlNet::ModType type);
+      void pointModified(ControlPoint *point, ControlPoint::ModType type, QVariant oldValue, QVariant newValue);
+      void measureModified(ControlMeasure *measure, ControlMeasure::ModType type, QVariant oldValue, QVariant newValue);
+      void pointDeleted(ControlPoint *point);
+      void newPoint(ControlPoint *);
+      void newMeasure(ControlMeasure *);
+      void measureRemoved(ControlMeasure *);
```

Comment:
Based on the other changes, emitNewMeasure and emitMeasureRemoved are no longer needed and can be removed